src/components/home/index.css

```diff
@@ -34,3 +34,152 @@
   text-align: justify;
   color: #ed2a7a;
 }
+
+.weCare {
+  color: black;
+  margin-left: 150px;
+}
+
+.weCare .pink {
```

Comment:
needs perhaps better class names, the classnames should capture what is contained in "pink". for example it's called pink but it has a lot of other properties besides being pink, i would expect something called .pink to just give it pinkness